semantics.json

```diff
@@ -113,6 +113,14 @@
     "importance": "low",
     "description": "These options will let you control the behaviour.",
     "fields": [
+      {
```

Comment:
When you change the structure of `semantics.json` or strings, please remember to also change the translation files accordingly. The H5P CLI tool has a utility command to `update-translations`. You can add a property `state` to a field in `semantics.json` with a value of `new`, `updated` or `removed` and the H5P CLI will change the structure/update labels accordingly. Moving fields around will involve manual changes to the translation files, however.

So, here you could have added `"state": "new"`, and running the `update-translations` command will add the respective fields to the translation files.

Reply:
Great tip. Thanks!

Reply:
Language files have now been updated.